src/test/cbmc/FatPointers/boxmuttrait.rs

```diff
@@ -11,4 +16,33 @@ fn main() {
     let buffer = vec![1, 2, 3, 5, 8];
     let num_bytes = log2.write(&buffer).unwrap();
     assert!(num_bytes == 5);
+
+
+    // Do some unsafe magic to check that we generate the right two vtables
+    unsafe {
+        let dest_trait_object: TraitObject = transmute(&*dest);
+        
+        // The vtable has [&drop, size, align, ....]
+        // The first 3 values are pointers, so we can grab them with offset
+        let dest_vtable_ptr = dest_trait_object.vtable as *mut usize;
+        let dest_size_ptr = dest_vtable_ptr.offset(1) as *mut usize;
+        let dest_align_ptr = dest_vtable_ptr.offset(2) as *mut usize;
```

Comment:
This code is repeated, maybe put it in a helper function?

Reply:
Done, shared across tests with `import!` as we discussed.